From 366e17553ed647613668678c2d301d369038f41b Mon Sep 17 00:00:00 2001
From: Brahadambal Srinivasan <latha@linux.vnet.ibm.com>
Date: Thu, 12 Nov 2020 19:00:47 +0530
Subject: [PATCH] Update ppc64-cpu usage

References: bsc#1183695 ltc#189235
Upstream: accepted - exected v1.3.9
Git-commit: 366e17553ed647613668678c2d301d369038f41b

'ppc64_cpu --help' doesn't list '--version' as an option. This patch
adds the option in the usage information of ppc64-cpu command.

Signed-off-by: Brahadambal Srinivasan <latha@linux.vnet.ibm.com>
Signed-off-by: Tyrel Datwyler <tyreld@linux.ibm.com>
---
 src/ppc64_cpu.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/ppc64_cpu.c b/src/ppc64_cpu.c
index 71f4720571b9..2b0f66cbbdbf 100644
--- a/src/ppc64_cpu.c
+++ b/src/ppc64_cpu.c
@@ -1195,7 +1195,8 @@ static void usage(void)
 "ppc64_cpu --subcores-per-core       # Get number of subcores per core\n"
 "ppc64_cpu --subcores-per-core=X     # Set subcores per core to X (1 or 4)\n"
 "ppc64_cpu --threads-per-core        # Get threads per core\n"
-"ppc64_cpu --info                    # Display system state information)\n");
+"ppc64_cpu --info                    # Display system state information\n"
+"ppc64_cpu --version                 # Display version of ppc64-cpu\n");
 }
 
 struct option longopts[] = {
-- 
2.26.2

